### PR TITLE
Organized table in bots.md and fixed a non-existent Installing Windows 10 article mention

### PR DESCRIPTION
**Pull request by itsWindows11**

### CODE REVIEW

File: about/bots.md [hunk 1]
```diff
@@ -38,9 +38,9 @@ You can find Dotsimus' website [here](https://dotsimus.com/). Additionally, you
 | Flareon             | [https://github.com/Ignaciox](https://github.com/Ignaciox/)                                           | Music                   |
 | Heartosoft          | [https://github.com/Microsoft-community/starboard](https://github.com/Microsoft-community/starboard)  | Starboard               |
 | iReport             | [https://discord.gg/apple](https://discord.gg/apple/)                                                 | Inter-server reporting  |
-| Microsoft Community | [http://mee6.xyz](http://mee6.xyz/)                                                                   | Levelling and utilities |
+| Microsoft Community | [http://mee6.xyz](http://mee6.xyz/)                                                                   | Leveling and utilities  |
```
Comment (Erisa) [inline about/bots.md:41]: Levelling is valid in British English, its not a typo

**Review by Erisa — CHANGES_REQUESTED** [review]

File: about/bots.md (hunk 1 above)
Comment (itsWindows11) [reply about/bots.md:41]: Maybe that's what being used to American English does to me lol. But I think some articles here are inconsistent in this area (usage of British vs American) so I don't really know if I should keep or revert this.

Comment (Erisa) [reply about/bots.md:41]: I would revert it for now pending a proper discussion about which English we want to use for the site, if we decide it necessary to be consistent

Comment (itsWindows11) [reply about/bots.md]: Alright, done.

**Review by Erisa — APPROVED** [review]

**Pull request merged**